### PR TITLE
tepl 6.5.1

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/tepl.rb [hunk 1]
```diff
@@ -22,13 +21,14 @@ class Tepl < Formula
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "amtk"
+  depends_on "gedit"
```
Comment (carlocab) [inline Formula/tepl.rb:24]: Oops. Can't do this since `gedit` depends on `tepl`.

**Comment by alebcay** [comment]
```
Run-time dependency libgedit-gtksourceview-300 found: NO (tried pkgconfig and framework)
```

```
News in 6.5.1, 2023-04-26
-------------------------
* Tepl now depends on libgedit-gtksourceview instead of GtkSourceView:
    https://github.com/gedit-technology/libgedit-gtksourceview
    https://gedit-technology.net/
* New classes: TeplStack, TeplStackItem and TeplStackSwitcherMenu.
* Remove the attic/ (no longer necessary with the TeplStack* classes).
* Build: adapt to the latest Meson version.
```

My guess is `gedit` was added due to the meson error output but as noted in changelog, `gedit` != `libgedit-gtksourceview`.

**Comment by p-linnane** [comment]
Do we need to create a `libgedit-gtksourceview` formula here?

**Comment by alebcay** [comment]
My thought is no, could just include it as a resource instead since it doesn't meet notability requirements and I've yet to see anything else need to depend on it (maybe worth checking if that's something that `gedit` etc. might need soon).

In any case, I took a look at the GitHub repo they linked and could not find a `300.x.x` tag, latest is `299.x.x`. So maybe it hasn't been pushed yet or is available elsewhere.

**Comment by p-linnane** [comment]
Looks like `gedit` will need it in the next release: https://gitlab.gnome.org/GNOME/gedit/-/commit/a6b6e0c0fc053cb234aa250d7030c4d5863ec51f

**Comment by p-linnane** [comment]
@alebcay While we're waiting for the `300.x.x` tag, what are your thoughts on `resource` versus formula?

**Comment by alebcay** [comment]
Since `gedit` will depend on `tepl` as well, my preference would be `libgedit-sourceview` as a formula. This way there will only be one copy of it in the dependency tree.

But open to others' thoughts on keeping it as a resource in the two formulae that need it (would probably need to keep them in sync), maybe there's some advantage there that I'm missing.

**Comment by carlocab** [comment]
Definitely package it as a separate formula. Bundling it as a resource makes it useless (or, at least, much harder to use) to users who might want to use it, and makes it harder for us to keep up-to-date with upstream releases. Both these considerations are, in my view, much more important than the notability requirement.

It's also important to remember that the notability requirement is there to keep out weekend projects that are likely to fall out of maintenance soon after being added to Homebrew, and this seems unlikely to apply to `libgedit-gtksourceview`.

**Comment by p-linnane** [comment]
Opened a first pass at `libgedit-gtksourceview`: https://github.com/Homebrew/homebrew-core/pull/129910

File: Formula/tepl.rb [hunk 2]
```diff
@@ -103,7 +102,7 @@ def install
       -lgobject-2.0
       -ltepl-6
       -lgtk-3
-      -lgtksourceview-4
+      -lgtksourceview
```
Comment (carlocab) [inline Formula/tepl.rb:105]:
```suggestion
      -lgedit-gtksourceview-300
```

**Review by carlocab — APPROVED** [review]

**Comment by fxcoudert** [comment]
This broke gedit: https://github.com/Homebrew/homebrew-core/issues/130859

**Comment by Frank-Zeyda** [comment]
Please also see:

https://gitlab.gnome.org/GNOME/gedit/-/issues/559

on the GNOME gedit project.

Stable gedit (44.2) appears not to be using tepl 6.5.1 yet according to the dev team there.

**Pull request merged**